index.html

```diff
@@ -89,8 +90,7 @@ <h6>Make a stunning website in minutes by using the Stone CSS framework</h6>
                 <hr>
                 <div id="right-footer">
                     <a href="#feature">Features</a>
-                    <a href="https://github.com/devarshishimpi/Stone-CSS-Website" target="_blank">Github</a>
-                    <a href="https://blog.stonecss.com" target="_blank">Blog</a>
+                    <a href="https://github.com/devarshishimpi/stonecss" target="_blank">Github</a>
```

Comment:
```suggestion
                    <a href="https://github.com/devarshishimpi/stonecss" target="_blank">Github</a>
                    <a href="https://blog.stonecss.com" target="_blank">Blog</a>
```
Please add the blog link here back too.